test/run-tests.js

```diff
@@ -205,6 +205,52 @@ function run_compress_tests() {
                             failed_files[file] = 1;
                         }
                     }
+                    // The test with expect_stdout passed.
+                    // Now try to reminify original input with standard options
+                    // to see if it matches expect_stdout.
+                    var reminify_input = input_code;
+                    var reminify_options = {
+                        output: {
+                            beautify: true,
+                        },
+                        mangle: false,
+                        compress: {
+                            passes: 9,
+                            toplevel: true,
+                            global_defs: options.global_defs,
```

Comment:
`global_defs` can be removed as it doesn't work with `expect_stdout` by definition.